src/components/Rubric/hooks.test.ts

```diff
@@ -0,0 +1,117 @@
+import { mocked } from 'ts-jest/utils';
+
+import {
+  usePageData,
+  useRubricConfig,
+} from 'data/services/lms/hooks/selectors';
+import { mockUseKeyedState } from '@edx/react-unit-test-utils';
+import { submitRubric } from 'data/services/lms/hooks/actions';
+import { useRubricData, stateKeys } from './hooks';
+
+jest.mock('data/services/lms/hooks/selectors');
+jest.mock('data/services/lms/hooks/actions');
+
+const mockedUsePageData = mocked(usePageData);
+const mockedUseRubricConfig= mocked(useRubricConfig);
+const mockedSubmitRubric = mocked(submitRubric);
+
+const states = mockUseKeyedState(stateKeys);
+
+describe('useRubricData', () => {
+  const mutateFn = jest.fn();
+
+  const mockPageData = {
+    rubric: {
+      optionsSelected: {
+        'criterion-1': 'option-1',
+        'criterion-2': 'option-2',
+      },
+      criterionFeedback: {
+        'criterion-1': 'feedback-1',
+        'criterion-2': 'feedback-2',
+      },
+      overallFeedback: 'overall-feedback',
+    },
+  }
+
+  mockedUsePageData.mockReturnValue(mockPageData as any);
+
+  mockedUseRubricConfig.mockReturnValue({
+    criteria: [
+      {
+        name: 'criterion-1',
+        options: [
+          { label: 'Option 1', value: 'option-1' },
+          { label: 'Option 2', value: 'option-2' },
+        ],
+      },
+      {
+        name: 'criterion-2',
+        options: [
+          { label: 'Option 1', value: 'option-1' },
+          { label: 'Option 2', value: 'option-2' },
+        ],
+      },
+    ],
+    feedbackConfig: {
+      enabled: true,
+    },
+  } as any);
+
+  mockedSubmitRubric.mockReturnValue({
+    mutate: mutateFn,
+  } as any);
+
+  test('state keys', () => {
```

Comment:
This is a very vague test name... should be explicit about what we are testing, which is state value initialization.
If in one test, should be something like `it('initializes state values from page data', () => {`.